notebooks/official/migration/UJ1 Vertex SDK AutoML Image Classification.ipynb

```diff
@@ -0,0 +1,1432 @@
+{
```

Comment:
<p>If you are going to call <code>wait</code>,  the cleaner way would be to just call <code>model.batch_predict</code>  with sync=false.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
I wanted to show an example of using wait(). I will think of some additional text to make clear.

Reply:
Makes sense!

Reply:
updated